src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -222,19 +222,18 @@ public virtual void ConfigureServices(IServiceCollection services)
                 }
                 else
                 {
-                    // TODO EES-5049 Admin needs user managed identity to access the public data database
-                    // services.AddDbContext<PublicDataDbContext>(options =>
-                    // {
-                    //     var sqlServerTokenProvider = new DefaultAzureCredential();
-                    //     var accessToken = sqlServerTokenProvider.GetToken(
-                    //             new TokenRequestContext(scopes:
-                    //                 ["https://ossrdbms-aad.database.windows.net/.default"]))
-                    //         .Token;
-                    //     var connectionStringWithAccessToken =
-                    //         publicDataDbConnectionString.Replace("[access_token]", accessToken);
-                    //     var dbDataSource = new NpgsqlDataSourceBuilder(connectionStringWithAccessToken).Build();
-                    //     options.UseNpgsql(dbDataSource);
-                    // });
+                    services.AddDbContext<PublicDataDbContext>(options =>
+                    {
+                        var sqlServerTokenProvider = new DefaultAzureCredential();
+                        var accessToken = sqlServerTokenProvider.GetToken(
+                                new TokenRequestContext(scopes:
+                                    ["https://ossrdbms-aad.database.windows.net/.default"]))
+                            .Token;
+                        var connectionStringWithAccessToken =
+                            publicDataDbConnectionString.Replace("[access_token]", accessToken);
```

Comment:
We actually need to supply Admin with the Public Data DB connection string in Azure when this goes in, so presumably this change will start to fail Admin starting up when it finds that it doesn't have an appropriate piece of configuration.

Given that Admin is controlled via the ARM template, we'll need to make the change in the ARM template within this PR I think.

We'll need to provide the connection string in the form:

`Server=<subscription>-ees-psql-flexibleserver.postgres.database.azure.com;Database=public_data;Port=5432;User Id=<subscription>-as-ees-admin;Password=[access_token]`

Reply:
This was already done by the ARM template change in https://github.com/dfe-analytical-services/explore-education-statistics/pull/4745/files. 🙂 

The connection string has been added as a key vault secret which contains the connection in the form you posted above.